lib/core/api/get-file.js

```diff
@@ -24,43 +27,98 @@ module.exports = function(self) {
     throw e;
   };
 
+  function createFileToken(file, cb) {
+    file._client.createToken(file._bucketId, 'PULL', function(e, body) {
```

Comment:
Not sure I follow. I think we are having a hard time defining the difference between the `File` class and the `Storj` class core components. I'm trying to maintain that all of the functions in the api directory are extensions of the `Storj` class and inherit its context so they can all work together.

So here we have `getFile` making calls to the core through the `File` class while the rest of the core components (api directory) are calling client through the `Storj` class. So this says to me that `getFile` should be moved entirely out of the core components and directly into the `File` class. In which case I would like to see a core api for `getStream` that has nothing to do with the `File` class. I wonder if we should plan for non-file related data since storj doesn't specifically store files.

My thoughts:

`Storj`: handles all core api calls (basically extending bridge client to the Storj class) and gives all the necessary tools a developer would need to interact with core including the nice top level commands that storj-cli provides. (we are already doing this with createFile). This has the potential of us just recreating storj-cli which already does all of this in node. So I think we should focus on the browser. So we ignore things in storj-cli that don't make sense in the browser. Storj.js = browser first. If you want node we already have tools for that.

`File`: is a convenience class if you are building an application that is centered around files and has its own api `renderTo, appendTo, storeFile, deleteFileFromStore...` . I don't think it should be the only way of getting data from the storage network in browser.  

I would like to see `storj.getData(bucketId, fileId)` or `getStream` to just return raw data.